ci/scripts/r_docker_configure.sh

```diff
@@ -30,6 +30,17 @@ fi
 # Ensure parallel compilation of C/C++ code
 echo "MAKEFLAGS=-j$(${R_BIN} -s -e 'cat(parallel::detectCores())')" >> $(R RHOME)/etc/Renviron.site
 
+# Figure out what package manager we have
+if [ "`which dnf`" ]; then
+  PACKAGE_MANAGER=dnf
+elif [ "`which yum`" ]; then
+  PACKAGE_MANAGER=yum
+elif [ "`which zypper`" ]; then
+  PACKAGE_MANAGER=zypper
+else
+  PACKAGE_MANAGER=apt-get
```

Comment:
Should we move the `apt-get update` up here from down in the `if` below? 

I suspect we don't have a test that has `ARROW_S3 == OFF` and `ARROW_R_DEV == FALSE`, but if we did the update wouldn't have happened there (line 66) so line 83 won't have the update yet

Reply:
yeah probably so